### PR TITLE
#155890684 Create the retrofit instance to make API requests

### PR DESCRIPTION
**Pull request by ongebo**
### What does this PR do?
Creates the Retrofit instance for making API requests
#### Description of the task to be completed?
- Add dependencies for Retrofit to app module's `build.gradle`
- Define retrofit instance in GithubService.java in a `service` subpackage
#### How should this be manually tested?
N/A
#### What are the relevant Pivotal Tracker stories?
[#155890684](https://www.pivotaltracker.com/story/show/155890684)

### CODE REVIEW

File: app/src/main/java/com/example/javacodersnairobi/service/GithubService.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.example.javacodersnairobi.service;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class GithubService {
+    private Retrofit retrofit;
+    private final String BASE_URL = "http://api.github.com/";
```
Comment (damafeez) [inline app/src/main/java/com/example/javacodersnairobi/service/GithubService.java:8]: Good job @ongebo, Don't you have to add the stack and country query strings to the URL?

Comment (ongebo) [reply app/src/main/java/com/example/javacodersnairobi/service/GithubService.java:8]: I think that should be added in the interface defining the API endpoints. Consider the endpoints below:

**Get all Users**
`https://api.github.com/search/users?q=type:User+location:Nairobi+language:JAVA%22`
**Get a single user**
`https://api.github.com/users/k33ptoo`

The base URL for the endpoints is: `http://api.github.com/`

Comment (damafeez) [reply app/src/main/java/com/example/javacodersnairobi/service/GithubService.java:8]: That's ok, I'll merge this in. @ongebo

**Review by damafeez — APPROVED** [review]
LGTM

**Pull request merged**